src/structures/MessageEmbed.js

```diff
@@ -62,9 +62,9 @@ class MessageEmbed {
 
     /**
      * The fields of this embed
-     * @type {EmbedField[]}
+     * @type {EmbedFieldData[]}
```

Comment:
Shouldn't the type stay the same? The type of the field doesn't change after all.

Reply:
I don't have a button to accept this change but yes, I think you're right.